framework/include/kernels/CoupledTimeDerivative.h

```diff
@@ -36,7 +36,6 @@ class CoupledTimeDerivative : public Kernel
   virtual Real computeQpJacobian();
   virtual Real computeQpOffDiagJacobian(unsigned int jvar);
 
-private:
   const VariableValue & _v_dot;
   const VariableValue & _dv_dot;
   unsigned int _v_var;
```

Comment:
Maybe this should be const if you are removing protection? Probably should have been before.